src/vm/mscorlib.h

```diff
@@ -752,10 +752,6 @@ DEFINE_CLASS(PARAMETER_MODIFIER,    Reflection,             ParameterModifier)
 
 DEFINE_CLASS(POINTER,               Reflection,             Pointer)
 
-DEFINE_CLASS_U(Reflection, Pointer, ReflectionPointer)
```

Comment:
Ok, thought I was just removing dead code - didn't realize what the "_U" signified.